interactors/repeatablefield.js

```diff
@@ -33,11 +30,10 @@ export const RepeatableField = HTML.extend('repeatable field')
     },
     removeDisabled: (el) => el.querySelector('[data-test-repeatable-field-remove-item-button]').disabled,
     itemCount: (el) => el.querySelectorAll('li').length,
-    headLabels: (el) => !!el.querySelector('[data-test-repeatable-field-list-item-labels]'),
   })
   .actions({
-    clickAddButton: ({ find }) => find(RepeatableFieldAddButton()).click(),
-    clickRemoveButton: ({ perform }, index) => {
+    clickAdd: ({ find }) => find(RepeatableFieldAddButton()).click(),
+    clickRemove: ({ perform }, index) => {
       return perform((el) => {
         const button = [...el.querySelectorAll('button[data-test-repeatable-field-remove-item-button]')].filter((b, i) => i === index)[0];
         return button ? button.click() : '';
```

Comment:
It just occured to me totally after the fact that if you made a `RepeatableFieldRemoveButton` with an `index` filter similar to the way we  did `MultiColumnList` https://github.com/folio-org/stripes-testing/blob/master/interactors/multi-column-list.js#L32

You could have the same API: `field.clickRemove(2)` but you would get much better error messaging and the JS might be a bit cleaner since it would look like:

```js
{
  //...
  clickRemove: ({ find }, index) => find(RepeatableFieldRemoveButton({ index })).click()
}